lib/matplotlib/widgets.py

```diff
@@ -271,9 +271,9 @@ class Slider(AxesWidget):
 
     Call :meth:`on_changed` to connect to the slider event
     """
-    def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt='%1.2f',
-                 closedmin=True, closedmax=True, slidermin=None,
-                 slidermax=None, dragging=True, **kwargs):
+    def __init__(self, ax, label, valmin, valmax, valstep=None, valinit=0.5,
+                 valfmt='%1.2f', closedmin=True, closedmax=True,
+                 slidermin=None, slidermax=None, dragging=True, **kwargs):
```

Comment:
Don't change the order of the arguments in an existing function, even if they are keyword arguments.

Reply:
What's the proper place to put them, before kwargs?

Reply:
yes